Sources/GraphQLWebSocket/Spec.swift

```diff
@@ -23,9 +23,6 @@ public enum ClientMessage: Encodable {
     ///         the server will close the socket with the event `4429: Too many initialisation requests`.
     case initialise(ConnectionInit)
     
-    /// A biderctional message used for detailing connection's health.
-    case ping(Ping), pong(Pong)
-    
```

Comment:
This is part of [WebSocket spec](https://datatracker.ietf.org/doc/html/rfc6455#section-5.5.2) and is handled by Starscream `WebSocket` as well as by `URLSessionWebSocketTask `.

I am not sure why `GraphQLWebSocket` tried to handle it itself, it used to send ping and pong frames with incorrect opcodes (which may work with some servers but its not following the spec)

Note that `GraphQLWebSocket` still keeps sending PINGs but it sends it with correct opcode (not as a binary message).